### PR TITLE
feature: remodel Composer sidebar for multibot

### PR DESCRIPTION
**Pull request by beyackle**
## Description

This redesigns the ProjectTree component to fit the new designs. The reveal/hide triangles work, although there's currently nothing anywhere but the Core Bot section, and it will re-render (for reasons yet unknown) when it gets revealed from being hidden.

Note: this is still a prototype; some functionality might be missing (like a11y).

## Task Item

refs #3466 

## Screenshots

image

### CODE REVIEW

**Comment by beyackle** [comment]
Updated to match the latest designs: 
image

**Comment by beyackle** [comment]
Putting this back into draft for a moment while I fix a bug; deleting something while we're looking at it makes the app crash.

**Comment by beyackle** [comment]
Couldn't reproduce that bug, but at least I fixed the problem with the headers' width.

File: Composer/packages/client/src/components/ProjectTree/treeItem.tsx [hunk 1]
```diff
@@ -38,6 +38,8 @@ const content = css`
   outline: none;
   display: flex;
   align-items: center;
+  justify-items: middle;
```
Comment (hatpick) [inline Composer/packages/client/src/components/ProjectTree/treeItem.tsx:41]: perhaps you meant 'center'
https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items

**Pull request merged**